src/components/KPop/KPop.vue

```diff
@@ -506,24 +509,24 @@ export default defineComponent({
 
     &:after {
       border-color: rgba(255, 255, 255, 0);
-      border-bottom-color: var(--KPopBackground, var(--white, color(white)));
+      border-bottom-color: var(--KPopBackground, var(--white, $tmp-color-white));
```

Comment:
question: why is this a temp token? Same question for all usage of `$tmp-color-white`

Reply:
Because we don't have a border color token for white and we can't go with a closest value we have token for.